cli.py

```diff
@@ -68,18 +71,35 @@ def main(log_level):
               type=str, help="D3A domain name")
 @click.option('-w', '--web-socket', default=DEFAULT_WEBSOCKET_DOMAIN,
               type=str, help="D3A websocket URL")
-@click.option('--run-on-redis', is_flag=True, default=False, help="Start the client using the Redis API")
-def run(setup_module_name, username, password, domain_name, web_socket, run_on_redis, **kwargs):
+@click.option('-i', '--simulation-info', type=str,
+              help="Simulation File info (accept absolute and relative path)")
+@click.option('--run-on-redis', is_flag=True, default=False,
+              help="Start the client using the Redis API")
+def run(base_setup_path, setup_module_name, username, password, domain_name, web_socket,
+        simulation_info, run_on_redis, **kwargs):
     if username is not None:
         os.environ["API_CLIENT_USERNAME"] = username
     if password is not None:
         os.environ["API_CLIENT_PASSWORD"] = password
     os.environ["API_CLIENT_DOMAIN_NAME"] = domain_name
     os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"] = web_socket
     os.environ["API_CLIENT_RUN_ON_REDIS"] = "true" if run_on_redis else "false"
+    if os.path.isabs(simulation_info):
+        os.environ["JSON_FILE_PATH"] = simulation_info
+    else:
+        os.environ["JSON_FILE_PATH"] = os.path.join(os.getcwd(), simulation_info)
 
     try:
-        importlib.import_module(f"d3a_api_client.setups.{setup_module_name}")
+        if base_setup_path is None:
+            importlib.import_module(f"d3a_api_client.setups.{setup_module_name}")
+        elif base_setup_path and os.path.isabs(base_setup_path):
+            sys.path.append(base_setup_path)
+            importlib.import_module(setup_module_name)
+        else:
+            setup_file_path = os.path.join(os.getcwd(), base_setup_path)
+            # print(f"pathhh: {pathhh}")
+            sys.path.append(setup_file_path)
+            importlib.import_module(setup_module_name)
```

Comment:
It could happen that the relative path for `setup_file_path` doesn't lead anywhere (it doesn't exist). In that case, a `ModuleNotFoundError` would be raised.
It would be good to catch this and show an explicative error to the user to tell them what went wrong.

Reply:
Thanks for the suggestion, will add it.

Reply:
https://github.com/gridsingularity/d3a-api-client/pull/94/files#diff-7648f5f4cce4bf01d02de43f8c5f6f2130b787baece0cf6338210496d8bdfcdcR106-R107